change-notes/1.26/analysis-javascript.md

```diff
@@ -58,3 +58,5 @@
 
 ## Changes to libraries
 * The predicate `TypeAnnotation.hasQualifiedName` now works in more cases when the imported library was not present during extraction.
+* The class `DomBasedXss::Configuration` has been deprecated, as it has been split into `DomBasedXss::HtmlInjectionConfiguration` and `DomBasedXss::JQueryHtmlOrSelectorInjectionConfiguration`. Unless specifically working with jQuery sinks, uses of that
+class should be replaced with `HtmlInjectionConfiguration`.
```

Comment:
Could we also point the reader towards our Xss.ql for how to use both configurations at once? That is after all how you would work around this deprecation with a customized   XSS query.